src/models/order.js

```diff
@@ -8,24 +8,27 @@ module.exports = (sequelize, DataTypes) => {
      * The `models/index` file will call this method automatically.
      */
     static associate(models) {
-      Order.belongsTo(models.User);
-      Order.belongsTo(models.Team);
+      Order.belongsTo(models.User, {
+        foreignKey: 'userId',
+        allowNull: false,
+      });
+      Order.belongsTo(models.Team, {
+        foreignKey: 'teamId',
+        allowNull: false,
```

Comment:
thoughts on allowing null here? given that we may want to store all orders into our DB, including ones that are not associated with a team

Reply:
Yeah that makes sense - I'll remove it.